features/step_definitions/networking.rb

```diff
@@ -1319,3 +1319,13 @@
   cb[cb_name] = @result[:response].split("\n").first
   logger.info "Node's active nmcli connection uuid is stored in the #{cb_name} clipboard as #{cb[cb_name]}"
 end
+
+Given /^I save multus pod on master node to the#{OPT_SYM} clipboard$/ do | cb_name |
```

Comment:
@asood-rh, I think you need to handle the case in which the user didn't specific the clipboard name, you need to default it to something.

Reply:
@pruan-rht If the clipboard variable name is omitted, the user would get an error as below:-
undefined method `to_sym' for nil:NilClass (NoMethodError)
      /home/asood/github-projects/verification-tests/features/step_definitions/networking.rb:1327:in `/^I save multus pod on master node to the(?: :(\S+))? clipboard$/'
      features/tierN/networking/network-policy.feature:2351:in `I save multus pod on master node to the clipboard'
Also script uses the variable for validation. It is required. Do you still need default?

Reply:
@pruan-rht 
Anyways, incorporated your feedback
https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3240/console

Reply:
@pruan-rht How does providing default value helps?

Reply:
@asood-rh since your step definition is using the regexp `OPT_SYM`, which mean `Optional Symbol`,  you give the user the choice of not putting in the clipboard name.  So you need to handle that situation when the user use the step `Given I save multus pod on master node to the clipboard` which is acceptable according to your step definition.

Reply:
@pruan-rht  Got it, I could also it required instead.

Reply:
@asood-rh no need, handling the optional `cb_name` will take care of it.  It's fine to leave as it's